SayTheirNames/Source/Model/Person.swift

```diff
@@ -109,7 +109,7 @@ struct Person: Decodable {
 
 extension Person: SearchResult {
     var image: UIImage? {
-        STNAsset.Image.userCircle.image
+        nil // STNAsset.Image.userCircle.image
```

Comment:
Is there a reason to have this `nil` here?

Reply:
The image asset is low-quality, and it's not necessary now since we have only one search result type - person. If it's nil here, the image won't be shown